spec/spec_helper.rb

```
@@ -25,7 +25,7 @@ module Features
     c.syntax = :expect
   end
 
-  config.fail_fast = true
```

In CI, fail fast is a better strategy. 
In development, the entire suite is executed seldom (before pushing) usually to confirm that nothing is broken. Also, in that case, fail fast seems to be better to get faster feedback.